src/startStopScene.ts

```diff
@@ -69,6 +69,27 @@ export async function startScene(lineStart?: number) {
         lineNumber--;
     }
 
+    // If current line contains: "def construct(" without "#" before it,
+    // then we insert a special comment line, and move the cursor to it
+    // (because `manimgl -se <lineNumber>` doesn't work on the "def construct(" line)
+    let idx = lines[lineNumber].indexOf("def construct(");
```

Comment:
Even though it would be sacrilegious to write `def    construct()` (many spaces between `def` and `construct`), people could technically do it, so maybe it's better to match for `\s+` in-between `def` and `construct`